packages/neuron-wallet/src/database/address/address-dao.ts

```diff
@@ -331,6 +331,13 @@ class AddressStore {
   }
 
   static changed() {
-    AddressDbChangedSubject.getSubject().next("Updated")
+    if (process.send) {
+      process.send({
```

Comment:
Could you please explain a bit how this `process.send` works and how it relates to the existing rxjs subjects?

Reply:
We have subscribed to messages from sub threads in the main process.
https://github.com/nervosnetwork/neuron/blob/eddb4e4b78053ba81288b9bd9416f99b52a4434f/packages/neuron-wallet/src/block-sync- renderer/index.ts#L95-L113

In the original implementation, we can get the Rx Subjects instance of the main process through `remote.require`, but since the `remote module` is no longer available in the child process, we can only communicate through the child process's `process.send`.

Reply:
I would suggest wrapping this global variable `process` into a class such as `MasterProcess`. This makes the variable more expressive and easier to maintain whenever needed, such as when you want to stub the master process in the automated tests or modify the behavior of the function `send`.

Reply:
> I would suggest wrapping this global variable `process` into a class such as `MasterProcess`. This makes the variable more expressive and easier to maintain whenever needed, such as when you want to stub the master process in the automated tests or modify the behavior of the function `send`.

Good idea.